docs/business-finder-links-based-facets.md

```diff
@@ -0,0 +1,40 @@
+# EU Exit Business Readiness finder links-based facets
+
+This document explains how the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder and retrieve content from different publishing apps.
+
+## How is this different from the user perspective?
+
+In most finders, search results get more granular as you select values in different facets (the number of results goes down). The business finder search results get more _broad_ as you select values in different facets (number of results goes up). In addition, search results only show content from one publishing app.
```

Comment:
Should this be "multiple publishing apps",  or should the paragraph be restructured, so the description of regular finders come first (number of results goes down, results from one publishing app), followed by the description of the business finder (number of results goes up, results from multiple publishing apps)?

Reply:
Oops.